src/main/java/com/fabriccommunity/thehallow/registry/HallowedBlocks.java

```diff
@@ -74,10 +74,12 @@
 
 public class HallowedBlocks {
 	// TODO Sort into categories
-	public static final Block TINY_PUMPKIN = register("tiny_pumpkin", new TinyPumpkinBlock(FabricBlockSettings.of(Material.PUMPKIN).strength(1.0F, 1.0F).sounds(BlockSoundGroup.LANTERN).build()), new Item.Settings().group(TheHallow.GROUP).food(new FoodComponent.Builder().hunger(2).saturationModifier(0.4f).build()));
+	public static final Block TINY_PUMPKIN = register("tiny_pumpkin", new TinyPumpkinBlock(FabricBlockSettings.copy(Blocks.PUMPKIN).strength(1.0F, 1.0F).sounds(BlockSoundGroup.LANTERN).build()), new Item.Settings().group(TheHallow.GROUP).food(new FoodComponent.Builder().hunger(2).saturationModifier(0.4f).build()));
```

Comment:
If we're overriding pretty much everything, why have it copy from instead of using the material>

Reply:
Because we're not overriding everything! Material, Hardness, Sound Group, Explosion Resistance and Light Level are all inherited, whereas before this PR, where we didn't copy from the vanilla variants, the blocks had no hardness and stuff. Copying from a vanilla block makes it much easier to not miss things like that, and keeps us from having to define all of them.